R/order_ep_cols.R

```diff
@@ -0,0 +1,298 @@
+      "simd2020v2_hscp2019_quintile",
+      "ur8_2022",
+      "ur6_2022",
+      "ur3_2022",
+      "ur2_2022",
+      "cluster",
+      "demographic_cohort",
+      "service_use_cohort",
+      "ipdc",
+      "spec",
+      "sigfac",
+      "cup_pathway",
+      "cup_marker",
+      "cij_marker",
+      "cij_start_date",
+      "cij_end_date",
+      "cij_pattype_code",
+      "cij_ipdc",
+      "cij_admtype",
+      "cij_adm_spec",
+      "cij_dis_spec",
+      "cij_pattype",
+      "cij_ppa",
+      "cij_delay",
+      "smr01_cis_marker",
+      "diag1",
+      "diag2",
+      "diag3",
+      "diag4",
+      "diag5",
+      "diag6",
+      "op1a",
+      "deathdiag1",
+      "deathdiag2",
+      "deathdiag3",
+      "deathdiag4",
+      "deathdiag5",
+      "deathdiag6",
+      "deathdiag7",
+      "deathdiag8",
+      "deathdiag9",
+      "deathdiag10",
+      "deathdiag11",
+      "death_location_code",
+      "post_mortem",
+      "place_death_occurred",
+      "death_board_occurrence",
+      "arth_date",
+      "asthma_date",
+      "atrialfib_date",
+      "cancer_date",
+      "cvd_date",
+      "liver_date",
+      "copd_date",
+      "dementia_date",
+      "diabetes_date",
+      "epilepsy_date",
+      "chd_date",
+      "hefailure_date",
+      "ms_date",
+      "parkinsons_date",
+      "refailure_date",
+      "congen_date",
+      "bloodbfo_date",
+      "endomet_date",
+      "digestive_date",
+      "arth",
+      "asthma",
+      "atrialfib",
+      "cancer",
+      "cvd",
+      "liver",
+      "copd",
+      "dementia",
+      "diabetes",
+      "epilepsy",
+      "chd",
+      "hefailure",
+      "ms",
+      "parkinsons",
+      "refailure",
+      "congen",
+      "bloodbfo",
+      "endomet",
+      "digestive",
+      "attendance_status",
+      "stay",
+      "conc",
+      "mpat",
+      "cat",
+      "tadm",
+      "adtf",
+      "admloc",
+      "oldtadm",
+      "disch",
+      "dischto",
+      "dischloc",
+      "discondition",
+      "nhshosp",
+      "stadm",
+      "adcon1",
+      "adcon2",
+      "adcon3",
+      "adcon4",
+      "reftype",
+      "commhosp",
+      "uri",
+      "op1b",
+      "dateop1",
+      "op2a",
+      "op2b",
+      "dateop2",
+      "op3a",
+      "op3b",
+      "dateop3",
+      "op4a",
+      "op4b",
+      "dateop4",
+      "clinic_type",
+      "no_paid_items",
+      "yearstay",
+      "jan_beddays",
+      "feb_beddays",
+      "mar_beddays",
+      "apr_beddays",
+      "may_beddays",
+      "jun_beddays",
+      "jul_beddays",
+      "aug_beddays",
+      "sep_beddays",
+      "oct_beddays",
+      "nov_beddays",
+      "dec_beddays",
+      "cost_total_net",
+      "cost_total_net_inc_dnas",
+      "jan_cost",
+      "feb_cost",
+      "mar_cost",
+      "apr_cost",
+      "may_cost",
+      "jun_cost",
+      "jul_cost",
+      "aug_cost",
+      "sep_cost",
+      "oct_cost",
+      "nov_cost",
+      "dec_cost",
+      "social_care_id",
```

Comment:
Could we please move `person_id` under `social_care_id`? i think that makes more sense as its a social care derived variable